README.md

```diff
@@ -58,7 +58,7 @@ In `button.md` file we can add:
 A simple button.
 ```
 
-Now, you can open [button](http://localhost:3000/atom.html#button) atom inside your browser (serve is needed).
+Now, in the atom tab, you can open [button](http://localhost:3000/atom/atom.html#button) atom inside your browser (serve is needed).
```

Comment:
I don't understand the meaning of `in the atom tab`.

You mean that we have to create a new tab in our web browser?

Reply:
I mean we have to go to the atom tab on the same web browser